### PR TITLE
Update answerfile_4.2_basic.txt.j2

### PR DESCRIPTION
**Pull request by mkalfon**
Adding OVN setup answers to use in GE environments.

### CODE REVIEW

**Comment by mkalfon** [comment]
@tareqalayan @myakove  @kobihk Can you please review it?

File: roles/ovirt-engine-setup/templates/answerfile_4.2_basic.txt.j2 [hunk 1]
```diff
@@ -66,6 +66,9 @@ OVESETUP_CONFIG/imageioProxyConfig=bool:True
 OVESETUP_PROVISIONING/postgresProvisioningEnabled=bool:True
 OVESETUP_APACHE/configureRootRedirection=bool:True
 OVESETUP_APACHE/configureSsl=bool:True
+OVESETUP_OVN/ovirtProviderOvn=bool:True
```
Comment (petr-balogh) [inline roles/ovirt-engine-setup/templates/answerfile_4.2_basic.txt.j2]: look at line number 20 this answer is already there so put it together there and omit this line, also I am for to put it to upgrade answer file if it's needed

Comment (mkalfon) [inline roles/ovirt-engine-setup/templates/answerfile_4.2_basic.txt.j2]: Thanks Petr, I overlooked it. Added additional patch.

Comment (mkalfon) [inline roles/ovirt-engine-setup/templates/answerfile_4.2_basic.txt.j2]: As for the upgrade, installing OVN using engine-setup is currently supported only in a clean installation state.

File: roles/ovirt-engine-setup/templates/answerfile_4.2_basic.txt.j2 [hunk 2]
```diff
@@ -18,6 +18,8 @@ OVESETUP_CONFIG/remoteEngineHostSshPort=none:None
 OVESETUP_CONFIG/fqdn=str:{{ ansible_fqdn }}
 OVESETUP_CONFIG/storageType=none:None
 OVESETUP_OVN/ovirtProviderOvn=bool:True
+OVESETUP_OVN/ovirtProviderOvnUser=str:{{ovirt_engine_db_user}}@internal
```
Comment (petr-balogh) [inline roles/ovirt-engine-setup/templates/answerfile_4.2_basic.txt.j2]: I would also cerate another vairable then ovirt_engine_db_user..   something more related with ovn and you can specify also its default value in defaults variables here: defaults/main.yml   something like ovirt_engine_ovn_user, same for password. I remember that when I run our upgrade job, answer OVESETUP_OVN/ovirtProviderOvn=bool:True was needed, so I needed add it there on line 15 as you can see. But it didn't ask about user and password.

Comment (mkalfon) [inline roles/ovirt-engine-setup/templates/answerfile_4.2_basic.txt.j2]: If you don't specify an answer? it takes the default?

Comment (petr-balogh) [inline roles/ovirt-engine-setup/templates/answerfile_4.2_basic.txt.j2]: I already installed 4.2 and it failed only cause of this answer were missing in answerfile OVESETUP_OVN/ovirtProviderOvn=bool , so I defined it and setup was OK. Don't know what are defaults values for user and password, didn't test it cause if didn't requir these answers from me.

File: roles/ovirt-engine-setup/templates/answerfile_4.2_basic.txt.j2 [hunk 3]
```diff
@@ -18,6 +18,7 @@ OVESETUP_CONFIG/remoteEngineHostSshPort=none:None
 OVESETUP_CONFIG/fqdn=str:{{ ansible_fqdn }}
 OVESETUP_CONFIG/storageType=none:None
 OVESETUP_OVN/ovirtProviderOvn=bool:True
+OVESETUP_OVN/ovirtProviderOvnPassword=str:{{ovirt_provider_ovn_password:}}
```
Comment (lukas-bednar) [inline roles/ovirt-engine-setup/templates/answerfile_4.2_basic.txt.j2]: I think there is extra colon at the end of variable.

**Pull request closed without merging**